apps/jam-ui/src/components/balance/queries.tsx

```diff
@@ -5,6 +5,7 @@ import { parseReportPayload } from "../../utils/rollups.inspect";
 import { InspectResponseBody } from "../../utils/rollups.types";
 
 const rollupHost = process.env.NEXT_PUBLIC_ROLLUPS_ENDPOINT;
+const dappAddress = process.env.NEXT_PUBLIC_DAPP_ADDRESS;
```

Comment:
@riseandshaheen, Would the `NEXT_PUBLIC_DAPP_ADDRESS` actually be the `NEXT_PUBLIC_APP_ADDRESS` that is defined in the .env? Or is that something new?

Reply:
You're right @brunomenezes! Fixed [here](https://github.com/comet-ing/comet/pull/10/commits/2cc946cda67669854d72dadfa74c82e02d323eaf)